ui/src/main/java/io/streamzi/eventflow/API.java

```diff
@@ -159,17 +160,21 @@ public void postFlow(String flowJson) {
     @GET
     @Path("/topics")
     @Produces("application/json")
-    public List<String> listTopicNames() {
+    public String listTopicNames() throws Exception {
```

Comment:
`/topics` implies there is a _collection_ - why do we need string ? 

Also the `throws`, I'd think we better catch stuff inside the method

Reply:
The other methods to do similar. I think we had problems with the escaping of a `List<String>` where each `String` has been created by `ObjectMapper#writeValueAsString`. The Strings were valid JSON but they got escaped on the wire for some reason.

Reply:
ok with me - thanks for clarification

Reply:
let's file an issue ? because ideally we use collection here, instead of long string, which may be a JSON array